### PR TITLE
Cleanup unused/project-specific code

### PR DESCRIPTION
**Pull request by weigao95**
For the cleanup described in #378. The fusion_server and its dependencies seem also need re-writing, but let's postpone it to the next PR.

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/spartan/381)

### CODE REVIEW

**Comment by gizatt** [comment]
jenkins please retest (you should be whitelisted now)

File: modules/spartan/director/iiwamanipdev.py [hunk 1]
```diff
@@ -1,123 +0,0 @@
-#system
```
Comment (manuelli) [inline modules/spartan/director/iiwamanipdev.py]: I would vote to keep this file, maybe in a re-named form. This is what sets up and launches the director visualizer and allows setting up custom extensions. Some of this functionality could be moved to the `iiwa_manip_app.py` file.

**Comment by manuelli** [comment]
LGTM with one minor comment above.

File: modules/spartan/director/iiwamanipdev.py (hunk 1 above)
Comment (weigao95) [reply modules/spartan/director/iiwamanipdev.py]: That file has been added back.

**Pull request merged**